.github/workflows/blog.yml

```diff
@@ -0,0 +1,26 @@
+name: Draft minor release blog post
+description: Drafts a new minor release blog post
+
+on: workflow_dispatch
```

Comment:
This means it runs when you trigger it manually from the Actions tab, right?

Reply:
yep!